tests/network/vmi_multus.go

```diff
@@ -335,26 +356,59 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 		})
 
 		Context("VirtualMachineInstance with Linux bridge plugin interface", func() {
-			table.DescribeTable("should be able to ping between two vms", func(interfaces []v1.Interface, networks []v1.Network, ifaceName string) {
+			getIfaceIPByNetworkName := func(vmiName, networkName string) (string, error) {
+				vmi, err := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Get(vmiName, &metav1.GetOptions{})
+				if err != nil {
+					return "", err
+				}
+
+				for _, iface := range vmi.Status.Interfaces {
+					if iface.Name == networkName {
+						return iface.IP, nil
+					}
+				}
+
+				return "", fmt.Errorf("couldn't find iface %s on vmi %s", networkName, vmiName)
+			}
+
+			generateIPAMConfig := func(ipamType string, subnet string) string {
+				return fmt.Sprintf("\\\"type\\\": \\\"%s\\\", \\\"subnet\\\": \\\"%s\\\"", ipamType, subnet)
+			}
+
+			table.DescribeTable("should be able to ping between two vms", func(interfaces []v1.Interface, networks []v1.Network, ifaceName, staticIPVm1, staticIPVm2 string) {
+				if staticIPVm2 == "" || staticIPVm1 == "" {
+					ipam := generateIPAMConfig("host-local", "10.1.1.0/24")
+					Expect(createBridgeNetworkAttachementDefinition("linux-bridge-net-ipam", 0, ipam))
```

Comment:
nit: you can inline these 2.

Reply:
I prefer not to because it will make the line to long I think.

I also forgot the `To(Succeed())` part , so I added it.